script/deb/deb-varnish-build.sh

```diff
@@ -7,12 +7,8 @@ SCRIPT_DIR=$(cd $(dirname $0); pwd)
 rm -rf ${VMP_ROOT_DIR}/src/debian
 cp -rp ${VMP_VARNISH_ORG_DIR}/pkg-varnish-cache/debian ${VMP_ROOT_DIR}/src/
 
-#Error when using symlink...
-FILES=`find ${VMP_ROOT_DIR}/src/debian/ -maxdepth 1 -type l | awk -F/ '{print $NF}'`
-for FILE in $FILES; do
-    rm ${VMP_ROOT_DIR}/src/debian/${FILE}
-    cp ${VMP_ROOT_DIR}/systemd/${FILE} ${VMP_ROOT_DIR}/src/debian/${FILE}
-done
+# resolve all the symlinks
+sed -i '' src/debian/varnish*
```

Comment:
An error occurs because the path to symlink does not exist.

It will be fixed after merging.
```
rm -rf ${VMP_ROOT_DIR}/src/debian
rm ${VMP_ROOT_DIR}/src/systemd
cp -rp ${VMP_VARNISH_ORG_DIR}/pkg-varnish-cache/debian ${VMP_ROOT_DIR}/src/
ln -s ${VMP_VARNISH_ORG_DIR}/pkg-varnish-cache/systemd ${VMP_ROOT_DIR}/src/systemd
# resolve all the symlinks
sed -i '' ${VMP_ROOT_DIR}/src/debian/varnish*
```